src/test/java/com/thindeck/dynamo/DyMemoTest.java

```diff
@@ -49,11 +54,49 @@
      */
     @Test
     public void read() throws IOException {
-        final DyMemo memo = new DyMemo("<memo/>");
+        final MkItem item = this.emptyMkItemWithMemo();
+        final DyMemo memo = new DyMemo(item);
         final XML xml = memo.read();
         MatcherAssert.assertThat(
             xml.node().getFirstChild().getLocalName(),
             Matchers.equalTo("memo")
         );
     }
+
+    /**
+     * Test {@link com.thindeck.dynamo.DyMemo#update(Iterable)}.
```

Comment:
please see http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html#test-method-names regarding test method comments and names